frontend/app/_components/ClientList.js

```diff
@@ -0,0 +1,39 @@
+import Client from "./Client";
+
+const clients = [
```

Comment:
I assume this is for testing purposes but as a rule you should not have these commited as part of production code. If you choose to do so, a good practice is to either have mock data in json files or at least name the variable mockdata or something:)

Reply:
Thank you for the feedback. Next time, we'll ensure the data is sourced directly from our database, as intended. You're absolutely right about the naming convention—I'll make sure to use descriptive names like mockData or similar in the future. I appreciate your guidance as I work to improve these practices.